savoirfairelinux/santropol-feast
# Rename "Diabetic Dessert" to "Diabetic" to work around #833.

## Works around #833 by shortening the name

### Status


### How to verify this change

*Fill out this section so that a reviewer can know how to verify your change.*

1. [Make a new order](http://127.0.0.1:8000/order/create/batch).
  - add 2 Diabetic Dessert
  - add 3 Green Salad
2. Go through the [Kitchen Count flow](http://127.0.0.1:8000/delivery/order/); confirm everything necessary
3. Print out the [route sheets](http://127.0.0.1:8000/delivery/routes/?print=yes)
  - you should see that your order is laid out like
    Diabetic   2
    Dessert   3
    Green Salad

Stop your docker instance, if applicable, pull the code, and put your docker instance back up if applicable. *Run the migration*: `cd src; python manage.py migrate`

Re-print the kitchen count. It should now be laid out like
    Diabetic   2
    Green Salad 3

### Deployment notes and migration


### New translatable strings


### Additional notes

This is not the final patch for #833; as I explained on that ticket, there's a deeper problem here. But it fixes the problem for now, and that's all we need.
